src/pages/books/RegisterBook.jsx

```diff
@@ -1,7 +1,105 @@
+import Form from '../../components/Form';
+import '../../styles/Registerbook.css';
+import InputLabel from '../../components/Label';
+import Header from '../../components/Header';
+import Input from '../../components/Input';
+import DatePickerComp from '../../components/DatePicker';
+import Button from '../../components/Button';
+import BasicSelect from '../../components/Select';
+import { collectBookData } from '../../services/book';
+import AlertComp from '../../components/Alert';
+import { useState } from 'react';
 const RegisterBook = () => {
+  const Categoria = ['Ficcion', 'Ciencia', 'Arte', 'Tecnologia', 'Ingenieria y Matematicas'];
+  const Idiomas = ['ES', 'ENG'];
+  const [alert, setAlert] = useState({ isOpen: false, message: '', severity: 'success' });
   return (
-    <div>
-      <h1>RegisterBook</h1>
+    <div className="ContMain">
+      <div className="ContHeader">
+        <Header TitlePage="Register Book"></Header>
+      </div>
+      <div className="ContForm">
+        <Form width="60%">
+          <div className="textFields1">
+            <InputLabel InputLabel="ISBN"></InputLabel>
+            <Input
+              height="44px"
+              width="100%"
+              colorHover="#394c73"
+              id="BookISBN"
+              placeholder="Enter ISBN"
+              type="text"
+              variant="outlined"
+            />
+            <InputLabel InputLabel="Title"></InputLabel>
+            <Input
+              height="44px"
+              width="100%"
+              colorHover="#394c73"
```

Comment:
MUI doen't have `colorHover` check the doc [Inputs](https://mui.com/material-ui/api/input/)